### ISSUE TITLE
could not import: pcre_free_study

### ISSUE THREAD

**Issue by baranskistad** [docopt/docopt.nim#31]
Whenever I try to run a script with docopt in it, this error appears:
```
could not import: pcre_free_study
```

Is this on my side or your side? I tried to install the package with nimble and it said it didn't exist.

**Comment by baranskistad** [comment]
This is my code
```
import os, docopt

let usage = """Usage: %s [options] [script.surf] [args]

Options:
    -h, --help     display help
    -v, --version  display version"""
let version = "1.0.0"
let args = docopt(usage, version = version)
```

**Comment by baranskistad** [comment]
Sorry, already solved #13
